core/modules/replica/CreateReplicaJob.h

```diff
@@ -145,7 +145,12 @@ class CreateReplicaJob
      *
      * @see Job::extendedPersistentState()
      */
-    std::list<std::pair<std::string,std::string>> extendedPersistentState() const override;
+    std::list<std::pair<std::string,std::string>> extendedPersistentState() const final;
```

Comment:
I'm pretty sure `final` does not imply `override` (it does seem to imply `virtual`, but that's not exactly the same...). Do you really mean to remove `override`?

Reply:
(I see other places you use `final` without override, I'm not going to mark it all up, just know that the question applies generally 😎 )

Reply:
According to the C++11 documentation:
```
In both cases, virt-specifier-seq, if used, is either override or final, or final override or override final
```


See: https://en.cppreference.com/w/cpp/language/override

Reply:
Also see these examples which contradict to you theory:
https://en.cppreference.com/w/cpp/language/final

Reply:
per our discussion; the functions are marked virtual in the base class. It makes sense to me now why this works.